src/server.js

```diff
@@ -23,3 +26,15 @@ export default (state: { page: string }): string => {
 
     return doc({ html, stylesForHead, state });
 };
+
+export default async (req: {}): string => {
+    if (req.url.includes('/pages/')) {
+        return renderPage(req.url.split('/pages/')[1]);
+    }
+    if (req.url.includes('/assets/javascript/')) {
+        return readFile(
+            path.join('dist', req.url.split('/assets/javascript/')[1]),
+            'utf8',
+        );
+    }
```

Comment:
A bit mad we have to implement our own `serve-static` middleware! Maybe `micro` is a bit too simple for this use case?

Reply:
would assets not be served from the CDN as normal?

Reply:
That's a good point too! I guess we just need to do this in our dev server

Reply:
Yeah, true. I guess this is fine as a first step, next we can look at how to deploy, where to host static assets.